src/js/holders/mixins/PropertyHolderMixin.ts

```diff
@@ -0,0 +1,84 @@
+import type { InMemoryEntity } from "@mat3ra/code/dist/js/entity";
+import { flattenObject } from "@mat3ra/code/dist/js/utils";
+import type { NameValueObjectExtended } from "@mat3ra/code/dist/js/utils/object";
+import type { PropertyHolderSchema } from "@mat3ra/esse/dist/js/types";
+
+import type { PropertyRowValue } from "../../Property";
+import PropertyFactory from "../../PropertyFactory";
+
+export type PropertyHolderSourceSchema = PropertyHolderSchema["source"];
+
+export type PropertyHolderMixin = {
```

Comment:
Here and below we may use the Pick statement like in https://github.com/Exabyte-io/standata/blob/epic/SOF-7613/src/js/types/application.ts#L3